server/index.js

```diff
@@ -1,22 +1,16 @@
 import http from 'http';
 import express from 'express';
-import mongoose from 'mongoose';
+import session from 'express-session';
+import passport from 'passport';
+import mongoose, { Mongoose } from 'mongoose';
 import bodyParser from 'body-parser';
 import cors from 'cors';
 import morgan from 'morgan';
-import colors from 'colors';
-import routes from './routes';
-import { config } from './config/config';
+import store from 'connect-mongo';
+import routes from './routes/index';
```

Comment:
You do not have to explicitly import the `index` file as it will be implicitly imported when import the folder;

Doing `import routes from './routes';` is just as good as doing `import routes from './routes/index';` See [here](https://github.com/airbnb/javascript#naming--filename-matches-export) for more insight

Reply:
thanks